tools/Cron_jobs/mailwatch.cron.monthly

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# MailWatch for MailScanner
+# Copyright (C) 2003-2011  Steve Freegard (steve@freegard.name)
+# Copyright (C) 2011  Garrod Alwood (garrod.alwood@lorodoes.com)
+# Copyright (C) 2014-2017  MailWatch Team (https://github.com/mailwatch/1.2.0/graphs/contributors)
+
+# GeoIP Update
+test -x /usr/local/bin/mailwatch_geoip_update.php || exit 0
+/usr/local/bin/mailwatch_geoip_update.php  >/dev/null 2>&1
```

Comment:
Give me a short code to do this if you have. I'll add it to the mailwatch.cron.monthly.